docs/tutorial/kymotracking.rst

```diff
@@ -481,187 +481,136 @@ Since this algorithm is specifically looking for curvilinear structures, it can
+        localization_variance_variance=ensemble_estimate.localization_variance_variance
+    )
 
-In the following, we'll use three simulated :class:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack` instances of length 62 based on a diffusion constant of `10.0`.
+For more information on this procedure and how it affects the estimation uncertainties, please refer to the :doc:`theory section on diffusive motion</theory/diffusion/diffusion>`.
 
-With Pylake, we can calculate the MSD from a :class:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack` with a single command::
+Mean Squared Displacements
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Mean Squared Displacements (MSD) are a different quantity used to characterize particle motion.
+Purely diffusive processes leads to Mean Squared Displacements that linearly depend on the lag time.
+MSDs for individual tracks are available through :meth:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack.msd`::
 
-    tracks[0].msd()
+    msd = tracks[0].msd()
 
-This returns a tuple of lags and MSD estimates. If we only wish MSDs up to a certain lag, we can provide a `max_lag` argument::
+Which returns a tuple of lags and MSD estimates. If we only wish MSDs up to a certain lag, we can provide a `max_lag` argument::
 
     >>> tracks[0].msd(max_lag = 5)
     (array([0.16, 0.32, 0.48, 0.64, 0.8 ]), array([ 3.63439512,  6.13181603,  9.08823918, 11.43574189, 12.61152129]))
 
-Similarly, we can compute an averaged MSD estimate for a number of tracks in a :class:`~lumicks.pylake.kymotracker.kymotrack.KymoTrackGroup` using :meth:`~lumicks.pylake.kymotracker.kymotrack.KymoTrackGroup.ensemble_msd`::
+If it is safe to assume that all particles exhibit the same diffusive motion, one can calculate the ensemble and time-averaged MSD for an entire group of tracks using the method :meth:`~lumicks.pylake.kymotracker.kymotrack.KymoTrackGroup.ensemble_msd`::
 
     ensemble_msd = tracks.ensemble_msd()
 
-This returns a weighted average of the MSDs coming from all the tracks (where the weight is determined by the number of points that contribute to each lag).
-Note that if the tracks are of equal length, this weighting will not have an effect (all the weights will be the same).
-The implicit assumption here is that all tracks are diffusive and their diffusion constants are the same.
-You can plot the ensemble msd as follows::
-
-    ensemble_msd.plot()
-
 The following properties can be read from :class:`~lumicks.pylake.kymotracker.detail.msd_estimation.EnsembleMSD`:
 
 - `lags` : Lags at which the MSD was computed.
 - `msd` : Mean MSD for each lag.
 - `sem` :  Standard error of the mean corresponding to each MSD.
 - `variance` : Variance of each MSD average.
 - `count` : Number of elements that contributed to the estimate at this lag.
-- `effective_sample_size` : Effective sample size.
+- `effective_sample_size` : Effective sample size used to compute the standard error of the mean.
```

Comment:
This is `EnsembleMSD` (e.g. the MSD estimates rather than the diffusion constant estimates based on the MSD estimates), not `EnsembleDiffusion`. Though, maybe revisiting it, maybe we should just constrain this to a link to the API docs (now that we have them) to keep the tutorial focused. I can add a line of text for the CVE as well.